src/appmixer/trello/checklist/AddChecklistToCard/component.json

```diff
@@ -0,0 +1,89 @@
+    "description": "Add a new checklist to a specific card.",
+    "private": false,
+    "version": "1.0.0",
+    "auth": {
+        "service": "appmixer:trello"
+    },
+    "quota": {
+        "manager": "appmixer:trello",
+        "resources": "AddChecklistToCard",
+        "scope": {
+            "userId": "{{userId}}"
+        }
+    },
+    "inPorts": [
+        {
+            "name": "in",
+            "schema": {
+                "type": "object",
+                "properties": {
+                    "boardId": { "type": "string" },
+                    "boardListCardId": { "type": "string" },
+                    "checklistName": { "type": "string", "maxLength": 16384 },
+                    "checklistItems": { "type": "string" }
+                },
+                "required": [ "boardListCardId", "checklistName" ]
+            },
+            "inspector": {
+                "inputs": {
+                    "boardId": {
+                        "type": "select",
+                        "label": "Board",
+                        "index": 1,
+                        "source": {
+                            "url": "/component/appmixer/trello/list/ListBoards?outPort=boards",
+                            "data": {
+                                "transform": "./transformers#boardsToSelectArray"
+                            }
+                        },
+                        "tooltip": "Select a board."
+                    },
+                    "boardListCardId": {
+                        "type": "select",
+                        "label": "Card",
+                        "index": 2,
+                        "source": {
+                            "url": "/component/appmixer/trello/list/ListBoardsCards?outPort=cards",
+                            "data": {
+                                "messages": {
+                                    "in/boardId": "inputs/in/boardId",
+                                    "in/isSource": true
+                                },
+                                "transform": "./transformers#cardListsToSelectArray"
+                            }
+                        },
+                        "tooltip": "Select a card or card ID from previous component."
+                    },
+                    "checklistName": {
+                        "type": "text",
+                        "label": "Checklist name",
+                        "index": 3
+                    },
+                    "checklistItems": {
+                        "type": "textarea",
+                        "label": "Checklist Items",
+                        "tooltip": "Enter up to 10 checklist items. Each item should be on a new line. If you need to add more items, you can do so after creating the card and using the AddChecklistItemToCard component.",
```

Comment:
it has "up to 10" , Create card has only 5, I may be better to sync this to up to 10 in both components